{"name": "test_login_user[17647352440-yy123456-False-200002-用户名或密码错误]", "status": "failed", "statusDetails": {"message": "AssertionError: 接口返回码是 【 200002 】, 返回信息：用户名或密码错误 \nassert False == 'False'", "trace": "self = <testcases.api_test.test_01_login_erp.TestUserLogin object at 0x0000020AB00EA8B0>\nbase_url = 'https://erp-qa.qdjk.xyz', username = '17647352440'\npassword = 'yy123456', except_result = 'False', except_code = '200002'\nexcept_msg = '用户名或密码错误'\n\n    @allure.story(\"用例--登录用户\")\n    @pytest.mark.parametrize(\"username, password, except_result, except_code, except_msg\",\n                             login_data[\"test_login_user\"])\n    def test_login_user(self,base_url, username, password, except_result, except_code, except_msg):\n        logger.info(\"*************** 开始执行用例 ***************\")\n        result = login_erp_user(base_url,username, password)\n        step_1(username)\n>       assert result.success == except_result, result.error\nE       AssertionError: 接口返回码是 【 200002 】, 返回信息：用户名或密码错误 \nE       assert False == 'False'\n\ntest_01_login_erp.py:25: AssertionError"}, "steps": [{"name": "步骤1 ==>> 登录用户", "status": "passed", "parameters": [{"name": "username", "value": "'17647352440'"}], "start": 1694412013474, "stop": 1694412013475}], "attachments": [{"name": "log", "source": "0b743351-e6b4-424b-b905-7ef017f83514-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "6525a944-65a5-4596-a348-5596d3612901-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "b315f03e-28a3-4315-841b-88658f85763b-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'17647352440'"}, {"name": "password", "value": "'yy123456'"}, {"name": "except_result", "value": "'False'"}, {"name": "except_code", "value": "'200002'"}, {"name": "except_msg", "value": "'用户名或密码错误'"}], "start": 1694412012772, "stop": 1694412013477, "uuid": "207aad26-a6dd-42cd-9102-28a8473d0c77", "historyId": "c34e0cd68573d7a4c0dd7563c1b60da5", "testCaseId": "80c254abc9876d28cea0e8148b92d53d", "fullName": "testcases.api_test.test_01_login_erp.TestUserLogin#test_login_user", "labels": [{"name": "epic", "value": "针对单个接口的测试"}, {"name": "severity", "value": "normal"}, {"name": "feature", "value": "用户登录模块"}, {"name": "story", "value": "用例--登录用户"}, {"name": "parentSuite", "value": "testcases.api_test"}, {"name": "suite", "value": "test_01_login_erp"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "WIN-FOCJ805QQO5"}, {"name": "thread", "value": "3744-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.api_test.test_01_login_erp"}]}